GCDWebUploader/GCDWebUploader.m

```diff
@@ -296,7 +296,13 @@ @implementation GCDWebUploader
 
 - (instancetype)initWithUploadDirectory:(NSString*)path {
   if ((self = [super init])) {
-    NSBundle* siteBundle = [NSBundle bundleWithPath:[[NSBundle mainBundle] pathForResource:@"GCDWebUploader" ofType:@"bundle"]];
+    NSString *bundlePath = [[NSBundle mainBundle] pathForResource:@"GCDWebUploader"
+                                                           ofType:@"bundle"];
+    if (!bundlePath) {
```

Comment:
OK I understand now. Nobody has ever asked for this however, and even then it's questionable that's the right approach. Think about it, now you have 2 copies of that bundle in the app, and it's very easy for them to get out of sync. The bundle is expected to be in sync with the code that uses it.

If you want to customize the CSS, it's much safer to fork the project. Please remove this commit.